Automattic/woocommerce-payments
# Add RTL support script for webpack

Fixes #

#### Add support for RTL CSS generation using a webpack config

This adds a new webpack config to generate rtl files to be used for RTL merchants. For every `style.css` file, a new one `style-rtl.css` will be generated. **This will raise our bundle size**.

Those files will be automatically registered and used when a merchant switches their store to an RTL language.









#### Testing instructions




Manual testing steps
* Without this PR, go to Settings > General in WP-Admin
* Switch the site language to Arabic, it should look like this:
<img width="312" alt="Screenshot 2023-08-02 at 15 07 59" src="https://github.com/Automattic/woocommerce-payments/assets/12542046/ba6e0ea4-fc0c-4aa1-9ff1-9736f314ca50">

* Then go back to Payments > Settings, you should see some problems like wrong padding, margins, etc. Something like this:

![slack-imgs](https://github.com/Automattic/woocommerce-payments/assets/12542046/60545bab-db74-4fa8-8a90-bd09cc580e88)


* Now, check this PR locally.
* Run `npm run dev`
* Refresh your page and double-check that all those margins and padding problems were fixed.
* Make a quick tour around the plugin pages and double-check if everything looks good.

-------------------


**Post merge**
